src/app/clusters/laundry-washer-controls-server/laundry-washer-controls-server.cpp

```diff
@@ -192,3 +192,53 @@ void MatterLaundryWasherControlsPluginServerInitCallback()
     LaundryWasherControlsServer & laundryWasherControlsServer = LaundryWasherControlsServer::Instance();
     registerAttributeAccessOverride(&laundryWasherControlsServer);
 }
+
+Status MatterLaundryWasherControlsClusterServerPreAttributeChangedCallback(const chip::app::ConcreteAttributePath & attributePath,
```

Comment:
This works for now, but should be replaced by more logic in attribute access interface logic if possible in a follow-up.

Reply:
Is there a code which has merged that we can reference? Thank you